lib/Config.php

```diff
@@ -162,7 +183,7 @@ public function get_connection($name)
 	 *
 	 * @return string
 	 */
-	public function get_default_connection_string()
+	public function get_default_connection_info()
```

Comment:
Yea, it'd be more semver friendly, and more of a warning to developers looking to upgrade. But yea, better off a call for @jpfuentes2